spark/dl/src/main/scala/com/intel/analytics/bigdl/utils/File.scala

```diff
@@ -209,6 +211,30 @@ object File {
       if (null != fs) fs.close()
     }
   }
+  def downloadFromUrl(url: String, dir: String): String = {
+    try {
+      val httpurl = new URL(url)
+      val fileName = getFileNameFromUrl(url)
+      System.out.println(dir + fileName)
+      val f = new File(dir + fileName)
+      FileUtils.copyURLToFile(httpurl, f)
+    } catch {
+      case e: Exception =>
+        e.printStackTrace()
+        return "Fault!"
+    }
+    "Successful!"
+  }
+
+  def getFileNameFromUrl(url: String): String = {
```

Comment:
private?

Reply:
I think java/scala should have functions to do this. Don't write it again by yourself